src/components/UI.jsx

```diff
@@ -193,12 +202,12 @@ const UI = ({ lineColor = 'blueGray', tiles, data, dataPath, lastModified }) =>
             <Chloropleth
               className='flex-grow'
               selected_area={areaState.loadingArea || areaState.currentArea}
-              tiles={tiles}
+              geojson={tiles}
+              config={tiles.config}
```

Comment:
Split out the config here because I reckon it might be better placed coming from the other config file. Happy to delay doing that though.